main.js

```diff
@@ -58,6 +60,18 @@ function checkLocalStorage() {
 }
 
 //form validation
+function checkDigits(event) {
+  var validCodes = [39, 37, 8, 9, 91, 13, 38, 40];
+
+  if (event.keyCode >= 48 && event.keyCode <= 57 || validCodes.includes(event.keyCode)) {
+    hideError(2)
+    hideError(3)
+    return
```

Comment:
The use of this return to break out of the function is really cool! I don't think the return on line 73 is necessary though.